merge-rfc.sh

```diff
@@ -1,41 +1,87 @@
-#!/usr/bin/env zsh
+#!/usr/bin/env bash
 
 set -euo pipefail
 
 script="$0"
+
+####
+# CONFIG
+#
+
+MAIN_BRANCH=${MAIN_BRANCH:-main}
+OWNER=${OWNER:-buildpacks}
+REPO=rfcs
+BOT_USERNAME="buildpack-bot"
+
+####
+# FUNCTIONS
+#
+
 usage() {
-  printf "Usage: %s [-i <issue>...] [-n] <PR#>\n" "$script"
+  printf "Usage: %s [-i <issue>...] [-n] <PR_NUMBER#>\n" "$script"
   printf "Flags:\n"
-  printf "  -i string\n\tgithub issue reference (example: buildpacks/spec#1)\n"
-  printf "  -n\n\tno issues\n"
+  printf "  -i string\n\tgithub issue reference (example: ${OWNER}/${REPO}#1)\n"
+  printf "  -n\n\tbe okay with no issues being created\n"
   exit 1
 }
 
-id() {
-  ID=$(find text -depth 1 | sed -E 's|^text/([[:digit:]]{4})-.*$|\1|' | sort | tail -n 1)
-  ((ID++))
-  printf "%04d" "${ID}"
+generate_id() {
+  id="$(find text -maxdepth 1 -type f | sed -E 's|^text/([[:digit:]]{4})-.*$|\1|' | sort | tail -n 1 | sed 's/^0*//')"
+  ((id++))
+  printf "%04d" "${id}"
 }
 
 link_issue() {
   local issue="$1"
-  link=$(echo "$issue" | sed -E 's|buildpacks/(.*)#([[:digit:]]+)|https://github.com/buildpacks/\1/issues/\2|')
+  link=$(echo "$issue" | sed -E "s|([^\/]+)/([^#]+)#([[:digit:]]+)$|https://github.com/\1/\2/issues/\3|")
```

Comment:
is this to accommodate forks?

Reply:
Yes, essentially. Something that I needed as I was developing and didn't see a specific reason to revert it.